docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
```

Comment:
we can remove `and merges`, fixing the issue is the 🔑 Merging is the end state of a PR, so that's a given 👍